x-pack/plugins/apm/public/components/app/tutorials/agent_config_table.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import type { ValuesType } from 'utility-types';
+import { get } from 'lodash';
+import { EuiBasicTable, EuiText, EuiBasicTableColumn } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+export function AgentConfigurationTable({
+  variables,
+  data,
+}: {
+  variables: { [key: string]: string };
+  data: {
+    apmServerUrl?: string;
+    secretToken?: string;
+    apiKey?: string;
+    apmServiceName: string;
+    apmEnvironment: string;
```

Comment:
Do we want to show the default values for service name and environment in the table?

Reply:
This was the existing behaviour at least for onPrem and Cloud instructions unless we want to change it?

Reply:
Yes

Reply:
I removed them from here, but i had to add it directly as they are also required not only for the code generation but table generation as well